test/e2e/getenvoy_run_test.go

```diff
@@ -23,10 +23,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/mholt/archiver/v3"
```

Comment:
some time in the future, I want to make a separate go.mod file for the e2e tests, so that this dep doesn't clutter main